Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.15 as builder
```

Comment:
This is OK, but `go.mod` also should be updated.

Reply:
an maybe in another PR

Reply:
I do not think that deserves a new PR, the reason of the change is the io.ReadAll change, so I would keep in this PR, but I'm happy to make a new PR with a line change :D (It's already on a different commit)

Reply:
It should be changed in the go.mod as well